### PR TITLE
feat: fetch registered project

### PR DESCRIPTION
**Pull request by xla**
Exposed the ability to fetch registered projects to check for existence. It currently is exposed for orgs, as the user part is waiting for radicle-dev/radicle-registry#389.

### CODE REVIEW

File: proxy/src/http/org.rs [hunk 1]
```diff
@@ -145,6 +192,47 @@ impl ToDocumentedType for registry::Org {
...
+            document::string()
+                .description("The id of the org")
+                .example("monadic"),
+        );
+        properties.insert(
+            "maybeProjectId".into(),
+            document::string()
+                .description("The id project attested in coco")
+                .example("monadic")
+                .nullable(true),
```
Comment (MeBrei) [inline proxy/src/http/org.rs:229]: The examples for the project name and project id should not be the same as the org id for clarification.

File: proxy/src/registry.rs [hunk 2]
```diff
@@ -43,25 +44,25 @@ pub struct Metadata {
...
     },
 
     /// Issue a user registration for a given handle storing the corresponding identity id.
     UserRegistration {
         /// Globally unique user handle.
-        handle: UserId,
+        handle: Id,
         /// Identity id originated from librad.
-        id: registry::String32,
+        id: Option<String>,
```
Comment (MeBrei) [inline proxy/src/registry.rs:65]: Is it allowed to register a user without this id?

Comment (xla) [reply proxy/src/registry.rs:65]: It's not required semantically by the spec

**Comment by rudolfs** [comment]
And another bug, when registering a project in an org the org name and project name are swapped in the response.

Screenshot 2020-04-30 at 10 17 13

**Pull request merged**